app/assets/stylesheets/local/faq.scss

```diff
@@ -0,0 +1,10 @@
+.faq-page {
+  dl {
+    dt {
+      margin-top: 1em;
+    }
+    dd {
+      margin-top: .25em;
+    }
+  }
+}
```

Comment:
is there a way to make this just house-style default for our DL's, instead of duplicating it between faq.scss and policy.scss?  Or maybe at least a common class in used in both for informational dt's or something? It's nice when everything is consistent on multiple pages, which these are at the moment via duplication, but would be nice if the code encouraged keeping them that way.